app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -991,3 +994,39 @@ fun getLocalizedStringForUserTab(ctx: Context, tab: UserTab): String {
     }
     return returnString
 }
+
+/**
+ * Returns localized Strings for ListingType Enum
+ */
+fun getLocalizedListingTypeName(context: Context, listingType: ListingType): String {
```

Comment:
Follow the convention above: `ctx`

Reply:
OK thanks for the heads up. I was looking for a convention to follow for naming, I found in Utils.kt context: Context instead of ctx was being used in convertSpToPx and getLocalizedSortinTypeName methods so I was following that. I'll make the fixes now and try to push them to this branch.